packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -139,6 +139,12 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 	@Input()
 	public tableRowsActionBarConfig: StarkActionBarConfig;
 
+	/**
+	 * function to generate a classNames for rows
```

Comment:
Typo: "generate a classNames" -> "generate classNames"

Reply:
There is another small typo: **F**unction to...